README.md

```diff
@@ -12,8 +12,26 @@ It requires:
 
 * Ruby 3.2.0
 * node 18.15.0
+
+[Installation guide](https://dev.trk.in.rs/)
```

Comment:
I was thinking to create a page on dev and link to it

Reply:
@duleorlovic Okay, you can discard this pull request then